flow/Knobs.h

```diff
@@ -43,22 +43,30 @@ enum class ConfigDBType {
 	PAXOS,
 };
 
+enum class Atomic { YES, NO };
+
 class Knobs {
 protected:
+	template <class T>
+	struct KnobValue {
+		T value;
```

Comment:
Nit: I think it would be more clear to make the `value` type `T*`, and let `T` be a non-pointer type.